### DISCUSSION TITLE
Sync two apps to play at the same time?

### DISCUSSION THREAD

**Discussion by longueurtv** [ExistentialAudio/BlackHole#738] [Q&A]
I'm trying to use BlackHole to send audio from notation software, Sibelius Ultimate, to Ableton 11. Is there a way to play both simultaneously so they sync up, similar to what ReWire used to do?

**Comment by aabstrkt** [comment]
https://youtu.be/5z9JGwOvYJw?si=x7H_D0x3AsjxNmO1

Gaia Live
1995 - 2023
- - - - -
***@***.***
***@***.***
07415201433

**Reply by longueurtv** [reply]
Thanks for the link! This is a little different than what I'm trying to do -- I'm trying to send Sibelius audio to Ableton. The connection is working, I hear Sibelius audio in Ableton, but I'd like for them to sync up. At the end of the video he says he doesn't know any way to sync the midi clocks, so maybe it's not possible now?

**Comment by aabstrkt** [comment]
If you’re using Ableton 10 or earlier you might try to slave Sibelius to
Ableton using Ableton’s Rewire mode …


Then in the transport of Sibelius you will see the Rewire icon



This used to work, but I myself am on Ableton 11 Suite now, and Rewire
functionality has been removed.

In this case I use the SYNC setting.. sometimes with ableton as master,
sometimes with an external hardware sequencer instead:

https://help.ableton.com/hc/en-us/articles/209071149-Synchronizing-Live-via-MIDI

Tim x

Gaia Live
1995 - 2023
- - - - -
***@***.***
***@***.***
07415201433
